### ISSUE TITLE
[BUG] - ollama run error

### ISSUE THREAD

**Issue by zhaopengme** [kaiban-ai/KaibanJS#97]
```

  // Define an agent
  const researchAgent = new Agent({
    name: 'Researcher',
    role: 'Information Gatherer',
    goal: 'Find relevant information on a given topic',
    background: '',

    llmInstance: new ChatOllama({
      model: "llama3.2:3b",
      temperature: 0,
      maxRetries: 2,
      // other params...
    })

  });

  // Create a task
  const researchTask = new Task({
    description: 'Research recent AI developments',
    agent: researchAgent,
    expectedOutput: ''
  });

  // Set up a team
  const team = new Team({
    name: 'AI Research Team',
    agents: [researchAgent],
    tasks: [researchTask],
  });


  // Start the workflow
  team.start()
    .then((output) => {
      console.log("Workflow completed:", output);
    })
    .catch((error) => {
      console.error("Workflow error:", error);
    });
```
![image](https://github.com/user-attachments/assets/fbfac3a7-d61e-47ab-af3f-30bcacf5c9da)

**Comment by darielnoel** [comment]
Thansk for letting us know. On it

**Comment by darielnoel** [comment]
Hi @zhaopengme,

Thank you for bringing this issue to our attention and sorry for the inconvenience. We're currently looking into the bug, but in the meantime, you can resolve the error by adding an empty tools array to your agent definition. Here’s how you can modify your agent configuration:

```javascript
// Define an agent
const researchAgent = new Agent({
  name: 'Researcher',
  role: 'Information Gatherer',
  goal: 'Find relevant information on a given topic',
  background: '',
  tools: [], // Add this line
  llmInstance: new ChatOllama({
    model: "llama3.2:3b",
    temperature: 0,
    maxRetries: 2,
    // other params...
  })
});
```

Please try this workaround and let us know if it resolves the issue. We’ll keep you updated on our progress fixing this bug!

Best,
Dariel

**Comment by darielnoel** [comment]
Hi @zhaopengme, the issue with the tools array has been fixed and is included in the current release, version 0.8.4. Please update to this latest version to apply the fix. Thanks for bringing this to our attention!

**Comment by darielnoel** [comment]
Awesome. Thanks again ;). Have you tried the UI?
